pkg/client/core/service_application_client_test.go

```diff
@@ -0,0 +1,132 @@
+package core_test
```

Comment:
This whole file looks like a nice example for others to follow when creating a service test :+1: